evm/README.md

```diff
@@ -197,9 +199,35 @@ $ anvil --help
 $ cast --help
 ```
 
-### Deploy Wormhole NTT
+## Message Customization
+
+See the [NttManager](../docs/NttManager.md) doc for wire format details.
+
+### NativeTokenTransfer Additional Payload
+
+Your contract can extend `NttManagerNoRateLimiting` to provide an additional payload on the `NativeTokenTransfer` message. Override the following:
+
+- `_prepareAdditionalPayload`
+- `_handleAdditionalPayload`
```

Comment:
It could be worth adding a note here to point out that `NttManager` may be extended if rate limiting is off, but the size of `NttManager` is significantly higher (and very close to the contract size limit) than `NttManagerNoRateLimiting`. This is pretty clear from the PR description but it's not clear to a reader of this README who doesn't have that context.